### ISSUE TITLE
TypeError: Cannot read property 'keyCode' of undefined

### ISSUE THREAD

**Issue by robertding** [daniel-nagy/md-data-table#228]
I use md-data-table with angular@1.4.8 and angular-material@1.0.1 with webpack.
when i click the md-checkbox on the top, it turned out the error.
```
TypeError: Cannot read property 'keyCode' of undefined
    at c.n.select (http://localhost:10010/bundle.js:107923:9978)
    at http://localhost:10010/bundle.js:107923:7972
    at Array.forEach (native)
    at Scope.c.selectAll (http://localhost:10010/bundle.js:107923:7930)
    at Scope.c.toggleAll (http://localhost:10010/bundle.js:107923:8048)
    at fn (eval at <anonymous> (http://localhost:10010/bundle.js:22318:16), <anonymous>:4:218)
    at callback (http://localhost:10010/bundle.js:32566:18)
    at Scope.$eval (http://localhost:10010/bundle.js:25005:29)
    at Scope.$apply (http://localhost:10010/bundle.js:25105:26)
    at HTMLElement.<anonymous> (http://localhost:10010/bundle.js:32571:24)(anonymous function) @ bundle.js:21473(anonymous function) @ bundle.js:18245Scope.$apply @ bundle.js:25110(anonymous function) @ bundle.js:32571defaultHandlerWrapper @ bundle.js:12299eventHandler @ bundle.js:12287
```
the directive i write here

```
<md-table-container>
    <table md-table md-row-select ng-model='selected' md-progress='promise'>
        <thead md-head>
            <tr md-row>
                <th md-column ng-repeat='header in headers'>{{ header }}</th>
            </tr>
        </thead>
        <tbody md-body>
            <tr md-row md-select='item' md-select-id='{{item.id}}' md-auto-select ng-repeat='item in items'>
                <td md-cell ng-repeat='header in headers'>{{ item[header] }}</td>
            </tr>
        </tbody>
    </table>
</md-table-container>
```

**Comment by robertding** [comment]
problem solved, it seems 0.9.4 on npm is not a good one, i use 0.8.11 instead

**Comment by daniel-nagy** [comment]
hmmm... sounds like something was going wrong when you compiled all your assets together with webpack.
